automated/utils/upload-to-squad.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+ATTACHMENT=""
+ARTIFACTORIAL_URL=""
+CURL_VERBOSE_FLAG=""
+FAILURE_RETURN_VALUE=0
+
+usage() {
+    echo "Usage: $0 [-a <attachment>] [-u <artifactorial_url>] [-t <artifactorial_token>] [-v] [-r]" 1>&2
```

Comment:
"t" option is missing from the help text.

Reply:
thanks for pointing out it! will drop the "t" description from here.
The idea here is to not support passing the token to avoid the possibility of leak.